Expecto.Differ/Expecto.Differ.fsproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <AssemblyName>Expecto.Differ</AssemblyName>
+        <PackageId>Expecto.Differ</PackageId>
+        <Version>8.12.0</Version>
+        <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <!-- Optional: Declare that the Repository URL can be published to NuSpec -->
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
I haven't seen this before? What's its effect?

Reply:
I just copied from Expecto.Halfo. I know almost nothing about these XML-files; i just really want them to work on my machine and for my IDE to recognize them :)  I'm basicly banging rocks together hoping for the depedencies to resolve.